Source/WebCore/rendering/svg/RenderSVGPath.cpp

```diff
@@ -67,7 +67,7 @@ void RenderSVGPath::updateShapeFromElement()
         m_shapeType = ShapeType::Path;
 }
 
-FloatRect RenderSVGPath::adjustStrokeBoundingBoxForMarkersAndZeroLengthLinecaps(FloatRect strokeBoundingBox) const
+FloatRect RenderSVGPath::adjustStrokeBoundingBoxForMarkersAndZeroLengthLinecaps(RepaintRectCalculation, FloatRect strokeBoundingBox) const
```

Comment:
I see, so it's because in LBSE we `RenderSVGShape::repaintRectInLocalCoordinates` calls `SVGBoundingBoxComputation::computeRepaintBoundingBox`, which includes markers through the functions you mention, rather than relying on a pre-computed repaint rect.

Reply:
Yeah, and I think this is very good direction! We were using repaint bonding box for non-repainting purpose, and LBSE is avoiding it.